packages/core/src/app/payment/paymentMethod/PaypalCommerceCreditDescription.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FunctionComponent, useEffect } from 'react';
+
+import { useCheckout } from '@bigcommerce/checkout/payment-integration-api';
+
+import PaymentMethodId from './PaymentMethodId';
+
+const PaypalCommerceCreditDescription: FunctionComponent<{ onUnhandledError?(error: Error): void }> = ({ onUnhandledError }) => {
+    const { checkoutService } = useCheckout();
+
+    useEffect(() => {
+        try {
+            checkoutService.initializePayment({
+                methodId: PaymentMethodId.PaypalCommerceCredit,
+                paypalcommercecredit: {
+                    // @ts-ignore
+                    bannerContainerId: 'paypal-commerce-banner-container',
+                },
+            });
+
+            checkoutService.deinitializePayment({
```

Comment:
Why do we deinitialize strategy immediately after initialization?

Reply:
Because we need to initialize it again when we select "Pay Later". This works the same way I did for Braintree. You can watch it here: https://github.com/bigcommerce/checkout-js/blob/master/packages/core/src/app/payment/paymentMethod/BraintreePaypalCreditDescription.tsx#L10